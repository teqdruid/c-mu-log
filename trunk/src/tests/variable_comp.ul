// test that is accepted by the parser but fails in the trans.ml
// variable comparison is not supported.

variable_comp($x,$y){
		     $X<$Y+1;
		    }	
