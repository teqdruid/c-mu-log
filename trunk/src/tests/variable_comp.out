Fatal error: exception Failure("Comparisons with multiple variables are unsupported.")
